sklearn/preprocessing/imputation.py

```diff
@@ -299,6 +312,22 @@ def _dense_fit(self, X, strategy, missing_values, axis):
 
             return most_frequent
 
+        # KNN
+        elif strategy == "knn":
+
+            if axis == 1:
+                X = X.copy().transpose()
+
+            full_data = X[np.logical_not(mask.any(1))]
+            if full_data.size == 0:
+                raise ValueError("There is no row with complete data!")
```

Comment:
Does the current documentation use the word "row"? I think I would rather use data point or sample.